corehq/apps/app_manager/models.py

```diff
@@ -5311,6 +5369,13 @@ def fetch_file(self, location):
 
         return location, content
 
+    def get_build_langs(self):
+        if self.build_profiles:
+            # return first profile, generated as part of lazy migration
+            return self.build_profiles[self.build_profiles.keys()[0]].langs
```

Comment:
I think `self.build_profiles.keys()[0]` has a non-deterministic result, and it definitely won't work on Python 3. See also http://stackoverflow.com/a/14959001/10840